Code/GraphMol/Wrap/props.hpp

```diff
@@ -77,13 +77,13 @@ boost::python::dict GetPropsAsDict(const T &obj,
   // std::vector<int>, std::vector<unsigned>, string
   STR_VECT keys = obj.getPropList(includePrivate, includeComputed);
   for(size_t i=0;i<keys.size();++i) {
-    if (AddToDict<double>(obj, dict, keys[i])) continue;
     if (AddToDict<int>(obj, dict, keys[i])) continue;
     if (AddToDict<unsigned int>(obj, dict, keys[i])) continue;
```

Comment:
I'm not sure what you mean by "boost figures out the signed unsigned types". The AddToDict<unsigned int> is only going to happen in cases when the int is larger than max signed. I don't think this is the idea.